### ISSUE TITLE
[NcActions] Inconsistent icon position

### ISSUE THREAD

**Issue by ShGKme** [nextcloud-libraries/nextcloud-vue#5284]
`#icon` slot is centered and `icon-class` is on top.

![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/25978914/5e91d9fc-1387-41c1-8490-219859c87ae1)

**Comment by ShGKme** [comment]
@nextcloud-libraries/designers what position is preferred?

**Comment by marcoambrosini** [comment]
Although I also prefer center aligned when there's only one line of text, I think that the better strategy for this component is to top align given that it accepts multiple lines of text. When there are multiple lines of text, the icon gets further and further away from the title.

Here's a real world example of top aligned. @ShGKme note that in your screenshot the icon is not properly centered with the text.

<img width="330" alt="Screenshot 2024-02-22 at 09 59 12" src="https://github.com/nextcloud-libraries/nextcloud-vue/assets/26852655/27044fef-4edc-4222-8c3d-1659d5d19584">

**Comment by jancborchardt** [comment]
Yep, confirming what @marcoambrosini said, it's better to have it aligned to the top, with the heading. Otherwise the icon is basically at an arbitrary position.

**Comment by ShGKme** [comment]
- Also InActionInput: https://github.com/nextcloud-libraries/nextcloud-vue/issues/6668

**Comment by jancborchardt** [comment]
Same thing with checkboxes in lists btw, FYI @mejo- @max-nextcloud:

Current state, vertically aligned | Would look more organized with `align-self: start`
-|-
![Image](https://github.com/user-attachments/assets/0460a764-d91c-42f9-859f-f5b094ad415a)|![Image](https://github.com/user-attachments/assets/0a61af6a-0dfc-4852-ba02-3d35873efada)

**Comment by nickvergessen** [comment]
@Antreesy @DorraJaouad

**Comment by Antreesy** [comment]
Talk would be quite an edge case. If you want to centralise it, let's decide on how it's supposed to look like in usual cases (for toggles and checkboxes it's a one-line message most of time, with rare occurences of wrapping into two lines). It's still a single interactive element with hover effects / background/ e.t.c. I personally can't say, which is better here

Centered | Top-align
-- | --
<img width="425" alt="Image" src="https://github.com/user-attachments/assets/9481ca7f-4c58-4407-a549-0c112d19ebba" /> | <img width="427" alt="Image" src="https://github.com/user-attachments/assets/e44cabdf-bfe9-4795-b68e-e3a785ddc47e" />

**Comment by ShGKme** [comment]
I'm voting for "always in top" 👀

**Comment by jancborchardt** [comment]
Pull request with fix at https://github.com/nextcloud-libraries/nextcloud-vue/pull/8730, please review. :)
